src/pdb/main.cpp

```diff
@@ -82,16 +83,18 @@ int main()
 
   printTimer.start();  // Start print timer right before entering infinite loop
 
+  bool emergencyButtonState = emergencyButton.read();
+
   while (1)
   {
     // Update EtherCAT variables
     ecat.update();
 
     // Get inputs from digitalIns and I2C bus
-    bool buttonstate = button.read();
+    bool onOffButtonstate = onOffbutton.read();
```

Comment:
I would change `onOffButtonstate` to `onOffButtonState` for consistency reasons. Also the build fails on `onOffbutton.read()` which should be changed to `onOffButton.read()` 

```suggestion
    bool onOffButtonState = onOffButton.read();
```